README.md

```diff
@@ -56,26 +56,30 @@ go get github.com/get-bridge/truss-cli truss
 With the `truss bootstrap` command, you can quickly and easily bootstrap your
 Truss deployment configuration using one of our pre-made templates. To get
 started, you'll need to configure the bootstrapper by creating a
-`bootstrap.truss.yaml` file. It should look something like this:
+`bootstrap.truss.yaml` file. It should look something like this, but be sure to
+fill out the `params` section as appropriate for your service:
 
 ```yaml
 # The following templateSource configuration represents the default values,
-# which means you'll need to have insopshub credentials loaded in order to
+# which means you'll need to have bridge AWS credentials loaded in order to
```

Comment:
I would just add a line that makes it a little more obvious as to what to run like:
```bash
$ aws-vault exec bridge -- truss bootstrap
```